src/python/src/lib/dist.py

```diff
@@ -22,11 +22,8 @@ def count_kmers(strings: Iterable, k: int) -> Counter:
 
 
 def kmerize_samples(samples: Mapping, k: int) -> dict:
```

Comment:
Вручную создают куски заданий, чтобы все данные за один раз прошли сериализацию. Если давать пулу самостоятельно создавать бэтчи, он будет делать это несколько раз, даже если указать большой размер батча. 

> On 20 May 2016, at 02:34, Nikolay Romashchenko <notifications@github.com> wrote:
> 
> In src/python/src/lib/dist.py:
> 
> > @@ -22,11 +22,8 @@ def count_kmers(strings: Iterable, k: int) -> Counter:
> >  
> >  
> >  def kmerize_samples(samples: Mapping, k: int) -> dict:
> Я пытался модифицировать этот код, чтобы он запустился, но логики с батчами не уловил. starmap же сам распоряжается тем, каким потокам раздать задания?
> К тому же batch оборачивается во что-то лишнее, см. ниже
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>